components/base/Banner.tsx

```diff
@@ -34,11 +34,11 @@ const StyledTopic = styled.div`
   font-weight: 200;
   line-height: 52px;
   text-align: center;
-  @media (${maxWidth(BREAKPOINT_IPAD)}) {
+  @media (${breakDown("sm")}) {
```

Comment:
This breaks downwards, with the rule starting to apply with the "sm" screen size.